ddl/placement/rule_test.go

```diff
@@ -137,7 +152,7 @@ func (t *testRuleSuite) TestNewRuleAndNewRules(c *C) {
 		name:     "zero count in object constraints",
 		input:    `{"+zone=sh,-zone=bj":0, "+zone=sh": 1}`,
 		replicas: 3,
-		err:      ErrInvalidConstraintsMapcnt,
+		err:      ErrMappingWrongSeparator,
```

Comment:
This is not a case of wrong separator, since `"..":4` is valid.

Reply:
Done